app/assets/stylesheets/styles/components/pages/_home.css.scss

```diff
@@ -68,58 +56,27 @@ video { display: block; }
   }
 
   h2 {
+    color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.